_po4a-tools/po4a-create-all-targets.sh

```diff
@@ -7,6 +7,11 @@
 # PUB_DIR directory to publish the localised files in
 # THRESHOLD translation % below which translated .md files are not generated
 
+# Get absolute path and CD to parent directory of script. Will work locally without it, but not via GH actions.
```

Comment:
This looks complicated. Where do you need to cd into?

../ ?

Maybe we can use GitHub actions variables?

Reply:
This was because for some reason, when you put the scripts in a folder (i.e. not in the root directory), they will work locally -changing the paths in them of course - but not when launched by a GH action. I presume because for some reason the script "thinks" it's being run from another directory, but don't ask me why... These lines get the absolute path for wherever the scripts are (locally or in a GH action) and cd's into the folder holding them, in this case "_po4a-tools", and runs the scripts from there. But... I'm just a monkey hammering at a keyboard, if you know a better way, great.

Reply:
Hmm yes. I think it’s a problem with the path you’re in if you run the script. If you’re not in the folder where the script is saved/root directory/whatever, the relative paths are wrong. 

Probably it’s similar to https://unix.stackexchange.com/questions/295806/how-to-execute-a-script-located-in-the-same-directory-as-the-current-script
and not easily solvable.

Reply:
Yes, when I was looking into it I found loads of examples, all using a similar command. This article explains what's going on, the title nicely condensing what I was struggling to explain: https://www.networkworld.com/article/2703490/unix---when-a-bash-script-asks--where-am-i--.html.

I've made a couple of changes, the variable name and the comment, hopefully making it a bit less cryptic.